share/pdudaemon.conf

```diff
@@ -160,6 +160,16 @@
         "aliastestdrive02": {
             "hostname": "drivealias",
             "port": 2
+        },
+        "groupalias01": {
+            "hostname": "http",
+            "port": 3,
+            "group": "testgroup"
+        },
+        "groupalias02": {
```

Comment:
Four ghe configuration would it make more sense to have something like (forgive me for broken syntax):
```
"groups": [ 
    testgroup: [{ "hostname": "h", "port", "p"}, { "hostname": "h2", ...}]
 ]
```
at the top level

having to create dummy names like `groupalias0` seems rather awkward both for config and for the code; And these do seem a different cartegory in their handling then normal aliases

Reply:
Those names were just for the test script. The only extra config is to an optional group to any existing aliases

Reply:
A so if you want ports as a group; you need to define aliases for them with a group name entry? That really wasn't clear to me thusfar sorry ;)

Reply:
Yes apologies it was not explained well. I've pushed a better config update that makes it clearer.